spring-xd-integration-test/src/main/java/org/springframework/xd/integration/util/StreamUtils.java

```diff
@@ -499,6 +499,26 @@ public static boolean waitForMetric(String name, URL adminServer, int waitTime)
 		return extractPidsFromJPS(response.getOutput());
 	}
 
```

Comment:
This looks like it was pulled in with some other unrelated changes under integration-test project